permissions/permissions_policy.csv

```diff
@@ -15,3 +15,9 @@ p, bb_user, data_public, read
 
 g, data_sensitive, data_private
 g, data_public, data_sensitive
+
+p, bb_user, data_public, label_write
+p, bb_user, data_public, label_i18n_write
+
+p, archive_label-moderation, data_public, label_read
+p, archive_label-moderation, data_public, label_moderate
```

Comment:
```
p, bb_user, data_public, label_read
p, bb_user, data_public, label_write
p, bb_user, data_public, label_i18n_write

p, archive_label-moderation, data_sensitive, label_read
p, archive_label-moderation, data_sensitive, label_write
p, archive_label-moderation, data_sensitive, label_i18n_write
p, archive_label-moderation, data_sensitive, label_moderate
```

also need to add above

```
p, archive_editor, data_sensitive, label_read
p, archive_editor, data_sensitive, label_write
p, archive_editor, data_sensitive, label_i18n_write
```

Reply:
I think something not clear for me, why need to give permission for users that dont must see it here?
1 p, bb_user, data_public, label_read - so I will need hide for them information about author (for on the future maybe will need show kc id or email)

2 why make all _data_sensitive_ for **archive_label-moderation** why not public? Its impossible to add label for cu that not public 

3 p, archive_label-moderation, data_sensitive, label_write - its impossible add label for cu from admin

4 p, archive_label-moderation, data_sensitive, label_i18n_write - you mean that usr can be **archive_label-moderation** but not **bb_user** ?

Reply:
1. I don't understand the problem. Try rephrasing.
2. data_sensitive includes data_public
3. For now, but api doesn't care for UI. AFAIK someone can send curl requests and it's ready for possible future with little effort.
4. No. Everybody in kc have bb_user.

'm trying to accomplish the following user stories:
* Simple user can see labels on public units
* Simple user can create new labels on public units. 
* Simple user can update the i18n of labels he created.
* archive_label-moderation user can do everything a simple user can do but also on sensitive units (not just public)
* archive_label-moderation user can moderate all labels. Sensitive and public units
* archive_editor can do everything archive_label-moderation user can do without moderation